core/string/node_path.cpp

```diff
@@ -330,6 +330,28 @@ NodePath NodePath::simplified() const {
 	return np;
 }
 
+NodePath NodePath::from_string_name(const StringName &p_string_name) {
+	String string = String(p_string_name);
+	const int size = string.size();
+	if (size == 0) {
+		return NodePath();
+	}
+	// Check if p_string_name contains a slash or colon. If so, we need to parse it.
+	const char32_t *chars = string.ptr();
+	for (int i = 0; i < size; i++) {
+		if (unlikely(chars[i] == '/' || chars[i] == ':')) {
```

Comment:
Makes me think it'd be interesting to see if we should add a `findmc` to complement `findmk`, and perhaps a `contains_any` or similar, there's many places that look for "is any of these present" and it'd be a good feature, just to think about I might look at an implementation later

Reply:
Wouldn't that require constructing a `Vector<char32_t>`, so less efficient than as it is now?

But in general, that seems like a good idea.